scheduler/src/cook/mesos/api.clj

```diff
@@ -1134,6 +1136,10 @@
       (empty? jobs)
       [true {:error "Need to specify at least 1 job."}]
 
+      (and (get-in ctx [:request :body-params :job])
+           (get-in ctx [:request :body-params :jobs]))
+      [true {:error "Can't specify both \"job\" and \"jobs\"."}]
```

Comment:
I suppose to be consistent we should also do that.  I can't think of a really good reason why someone would want to specify job UUID's in multiple ways...  so, yes, I'll add checks to make sure it's only specified in 1 of the 3 possible ways